notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        def seriarize_constants(self):
</pre><p>I think there's a typo in the function name here. It's supposed to be  <code>serialize_constants</code> instead of <code>seriarize_constants</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Oh good catch. Thanks</p>